app/controllers/questions_controller.rb

```diff
@@ -1,2 +1,16 @@
 class QuestionsController < ApplicationController
+  def new
+    @question = Question.new
+  end
+
+  def create
+    Question.create
```

Comment:
パラメータを渡さないと何も保存されませんよ。

```suggestion
    Question.create!(question_params)
```